src/routers/companyRouter.ts

```diff
@@ -46,15 +47,19 @@ router.get(config.CONFIRM_URL, async (req: Request, res: Response, next: NextFun
 });
 
 router.post(config.CONFIRM_URL, async (req: Request, res: Response, next: NextFunction) => {
-  const companyProfile: CompanyProfile | undefined = req.session?.getExtraData("companyProfile");
+  const companyProfile: CompanyProfile | undefined = req.session?.getExtraData(constants.COMPANY_PROFILE);
   if (companyProfile !== undefined) {
-    req.session?.setExtraData("companyNumber", companyProfile.companyNumber);
+    req.session?.setExtraData(constants.COMPANY_NUMBER, companyProfile.companyNumber);
```

Comment:
```suggestion
    req.session?.setExtraData(COMPANY_NUMBER, companyProfile.companyNumber);
```

Reply:
done - replaces all const.* statements with the actual constants